### PR TITLE
Feat toolbar sort instance modal list

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
Tracking Issue: https://github.com/ansible/awx/issues/3355

* Add sort to Instance Modal list
* Fix instance modal pagination bug
* Update url with instance and instance modal query params
* Remove `labelLink: '@'` now that we no longer use the binding in the project

##### ISSUE TYPE
 - Enhancement Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION

```
awx: 3.0.1
```

##### ADDITIONAL INFORMATION
instance modal

### CODE REVIEW

File: awx/ui/client/src/instance-groups/instances/instance-modal.partial.html [hunk 1]
```diff
@@ -18,16 +18,21 @@ <h3 class="at-Panel-headingTitle">{{ vm.panelTitle }} | {{ vm.instanceGroupName}
...
                     search-bar-full-width="true">
                 </smart-search>
             </div>
+            <at-list-toolbar
+                ng-if="vm.instances.length > 0"
+                sort-only="true"
+                sort-value="vm.toolbarSortValue"
+                sort-options="vm.toolbarSortOptions"
+                on-sort="vm.onToolbarSort">
+          </at-list-toolbar>
```
Comment (jlmitch5) [inline awx/ui/client/src/instance-groups/instances/instance-modal.partial.html]: needs to be tabbed over

**Review by jlmitch5 — APPROVED** [review]
looks great, just the one tabbing issue

File: awx/ui/client/src/instance-groups/instances/instance-modal.partial.html (hunk 1 above)
Comment (marshmalien) [reply awx/ui/client/src/instance-groups/instances/instance-modal.partial.html]: whoops. thanks!

**Review by elyezer — COMMENTED** [review]
LGTM, thank you for getting this done.

**Review by elyezer — APPROVED** [review]
LGTM, thank you for getting this done.

**Pull request merged**